package-lock.json

```diff
@@ -1059,12 +1059,6 @@
         "ansi-wrap": "0.1.0"
```

Comment:
This file shouldn't appear in this PR

Reply:
Im going to close this PR and redo the changes